src/pyflex/window_selector.py

```diff
@@ -306,6 +321,14 @@ def calculate_preliminiaries(self):
             self.peaks = self.peaks[(self.peaks > min_trough) &
                                     (self.peaks < max_trough)]
 
+    def print_remaining_windows(self):
```

Comment:
Yes I think that would make sense. Best just implement a `__str__` method for the window object. Then you can simply `print(self.windows)` for the same effect.